app/Http/Controllers/PagesController.php

```diff
@@ -28,8 +28,11 @@ public function contact()
         return view('contact');
     }
 
-    public function galerie()
+    public function prestations()
     {
-        return view ('galerie');
+        $mariages = Photo::select('image')->where('categories_id', 1)->get()->toArray();
```

Comment:
What happens if I delete the "mariages" category, and re-create one with the same name later ?
The ID will change, your code will be lost